package-lock.json

```diff
@@ -1,4884 +1,8 @@
 {
   "name": "jitsi-meet-electron-utils",
   "version": "3.0.0",
-  "lockfileVersion": 2,
```

Comment:
Please undo this change, aka, run npm install with npm 8

Reply:
Done